### PR TITLE
Cloud integration 

### PR DESCRIPTION
**Pull request by nevalla**

### CODE REVIEW

File: server/Gemfile [hunk 1]
```diff
@@ -9,6 +9,7 @@ gem 'mongoid', '~> 4.0.0'
 gem 'symmetric-encryption'
 gem 'activesupport', '~> 4.1.14'
 gem 'faye-websocket', '~> 0.10.0'
+gem 'eventmachine', '>= 0.12.0'
```
Comment (jakolehm) [inline server/Gemfile]: why?

Comment (nevalla) [inline server/Gemfile]: `CloudWebsocketConnectJob` and `Cloud::WebSocketClient` are using it. Should we get rid of it or just rely on `faye-websocket` will include it?

Comment (jakolehm) [inline server/Gemfile]: Server has been using eventmachine from the beginning.. I just thought that 0.12.0 version is somewhat misleading here.

Comment (nevalla) [inline server/Gemfile]: Okay, just randomly selected the same as faye-websocket is using. But let's remove that dependency as redundant(?))

File: server/app/jobs/cloud_websocket_connect_job.rb [hunk 2]
```diff
@@ -0,0 +1,51 @@
+require 'celluloid'
+require_relative '../services/logging'
+
+class CloudWebsocketConnectJob
+  include Celluloid
+  include Logging
+  include ConfigHelper # adds a .config method
+
+  def initialize(perform = true)
+    @root_url = ENV['CLOUD_WS_URL']
```
Comment (jakolehm) [inline server/app/jobs/cloud_websocket_connect_job.rb]: This env should be passed as a parameter.

**Comment by nevalla** [comment]
@jakolehm improved websocket client so that config/seed.yml reads socket uri from env variable and initializer script use that to configure client. Any better that way?

**Pull request merged**